packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -479,24 +479,43 @@ export default class RestApi {
 
       res.json(result)
     })
-    api.patch(
-      '/:collection/:object',
-      json(),
-      wrap(async (req, res) => {
-        const obj = req.xapiObject
-
-        const promises = []
-        const { body } = req
-        for (const key of ['name_description', 'name_label']) {
-          const value = body[key]
-          if (value !== undefined) {
-            promises.push(obj['set_' + key](value))
+    api
+      .patch(
+        '/:collection/:object',
+        json(),
+        wrap(async (req, res) => {
+          const obj = req.xapiObject
+
+          const promises = []
+          const { body } = req
+
+          for (const key of ['name_description', 'name_label', 'tags']) {
+            const value = body[key]
+            if (value !== undefined) {
+              promises.push(obj['set_' + key](value))
+            }
           }
-        }
-        await promises
-        res.sendStatus(204)
-      })
-    )
+
+          await promises
+          res.sendStatus(204)
+        })
+      )
+      .delete(
+        '/:collection/:object/tags/:tag',
+        wrap(async (req, res) => {
+          await req.xapiObject.$call('remove_tags', req.params.tag)
+
+          res.sendStatus(204)
+        })
+      )
+      .put(
```

Comment:
IMO, you should add a route to get tags.
`api.get('/:collection/:object/tags')`

Reply:
Tags are already available on the object itself, I'd like to avoid introducing unnecessary routes.